linode/obj/schema_resource.go

```diff
@@ -82,6 +82,11 @@ var resourceSchema = map[string]*schema.Schema{
 		Optional:    true,
 		Computed:    true,
 	},
+	"endpoint": {
+		Type:        schema.TypeString,
+		Description: "The endpoint for the bucket used for s3 connections.",
+		Computed:    true,
```

Comment:
We might want to also make this field optional so users can override the target endpoint.

Reply:
this would mean you could set the field in hcl and it would show up in the docs?

Reply:
You'll still need to manually add it to the docs, but making it both optional and computed will allow the endpoint to be overridden if specified.